draft-trammell-rains-protocol.md

```diff
@@ -170,14 +170,15 @@ In addition, the following terms are used in this document as defined:
 
 - Subject: A name or address about which Assertions can be made.
 - Object: A type/value pair of information about a name within an Assertion.
-- Assertion: A mapping between a Subject and an Object, signed by the Authority
-  for the namespace containing that Subject. See {{assertions}}.
+- Singular Assertion: A mapping between a Subject and one or several Objects,
```

Comment:
We still need a definition of "Assertion" as a superclass of these.

Reply:
It is described after all Sections are introduced. Should we move it up